CRITICAL - 2023-04-24 13:39:45 --> Undefined variable $personas
in APPPATH\Views\libros\listar.php on line 13.
 1 APPPATH\Views\libros\listar.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $personas', 'C:\\xampp\\htdocs\\Ejercicio06\\app\\Views\\libros\\listar.php', 13)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Ejercicio06\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Home.php(9): view('libros/listar')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-04-24 13:43:25 --> Undefined variable $datos
in APPPATH\Views\libros\listar.php on line 13.
 1 APPPATH\Views\libros\listar.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\Ejercicio06\\app\\Views\\libros\\listar.php', 13)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Ejercicio06\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Home.php(9): view('libros/listar')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-04-24 13:43:27 --> Undefined variable $datos
in APPPATH\Views\libros\listar.php on line 13.
 1 APPPATH\Views\libros\listar.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\Ejercicio06\\app\\Views\\libros\\listar.php', 13)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Ejercicio06\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Home.php(9): view('libros/listar')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-24 14:08:40 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'mibaselba' in C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'usuario01', Object(SensitiveParameterValue), 'mibaselba', 3306, '', 0)
#1 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Ejercicio06\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Ejercicio06\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Ejercicio06\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Ejercicio06\app\Controllers\Personas.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(934): App\Controllers\Personas->index()
#8 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
#9 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Ejercicio06\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'mibaselba' in C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Ejercicio06\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Ejercicio06\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Ejercicio06\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\Ejercicio06\app\Controllers\Personas.php(12): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(934): App\Controllers\Personas->index()
#7 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
#8 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Ejercicio06\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-24 14:08:40 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'mibaselba'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `persona`
ORDER BY `ci`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Personas.php(12): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Personas->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-24 14:08:41 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'mibaselba' in C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'usuario01', Object(SensitiveParameterValue), 'mibaselba', 3306, '', 0)
#1 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Ejercicio06\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Ejercicio06\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Ejercicio06\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Ejercicio06\app\Controllers\Personas.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(934): App\Controllers\Personas->index()
#8 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
#9 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Ejercicio06\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'mibaselba' in C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Ejercicio06\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Ejercicio06\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Ejercicio06\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\Ejercicio06\app\Controllers\Personas.php(12): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(934): App\Controllers\Personas->index()
#7 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
#8 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Ejercicio06\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-24 14:08:41 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'mibaselba'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `persona`
ORDER BY `ci`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Personas.php(12): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Personas->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-04-24 14:28:44 --> syntax error, unexpected token "}", expecting ";"
in APPPATH\Controllers\Personas.php on line 22.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Ejercicio06\\app\\Controllers\\Personas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Personas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Personas')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Personas', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-24 15:45:39 --> mysqli_sql_exception: Duplicate entry '9526652' for key 'PRIMARY' in C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Ejercicio06\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\Ejercicio06\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\Ejercicio06\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\Ejercicio06\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Ejercicio06\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Ejercicio06\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Ejercicio06\app\Controllers\Personas.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(934): App\Controllers\Personas->guardar()
#9 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
#10 C:\xampp\htdocs\Ejercicio06\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Ejercicio06\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-04-24 15:45:39 --> Duplicate entry '9526652' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`ci`, `nombre_com`, `fecha_nac`, `telefono`, `departamento`) VALUES (\'9526652\', \'Pedro\', \'2000/05/05\', \'591321654\', \'sc\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`ci`, `nombre_com`, `fecha_nac`, `telefono`, `departamento`) VALUES (\'9526652\', \'Pedro\', \'2000/05/05\', \'591321654\', \'sc\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`ci`, `nombre_com`, `fecha_nac`, `telefono`, `departamento`) VALUES (\'9526652\', \'Pedro\', \'2000/05/05\', \'591321654\', \'sc\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`ci`, `nombre_com`, `fecha_nac`, `telefono`, `departamento`) VALUES (:ci:, :nombre_com:, :fecha_nac:, :telefono:, :departamento:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Personas.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Personas->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personas))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
